File: 151003/Harlap/task1/src/main/java/by/harlap/rest/dto/response/AuthorResponseDto.java
package by.harlap.rest.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AuthorResponseDto {

    private Long id;
    private String login;
    private String password;
    private String firstname;
    private String lastname;
}

File: 151003/Yagnnish/Lab4/Dtos/ResponseDtos/AuthorResponseDto.cs
namespace lab_1.Dtos.ResponseDtos
{
    public class AuthorResponseDto : BaseResponseDto
    {
        public string? Login { get; set; }
        public string? Password { get; set; }
        public string? Firstname { get; set; }
        public string? Lastname { get; set; }

 
    }
}
